components/modal/demo/locale.md

```diff
@@ -1,10 +1,18 @@
 ---
 order: 6
-title: 国际化
+title: 
+  zh-CN: 国际化
+  en-US: Internationalization
 ---
 
+## zh-CN
+
 设置 `okText` 与 `cancelText` 以自定义按钮文字。
 
+## en-US
+
+To customize the text of buttons via set `okText` and `cancelText`.
```

Comment:
To customize the text of the buttons, you need to set `okText` and `cancelText` props.